exotica_core/src/kinematic_tree.cpp

```diff
@@ -1014,6 +1052,35 @@ void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
     UpdateJointLimits();
 }
 
+void KinematicTree::SetFloatingBaseLimitsPosXYZEulerZYX(
+    const std::vector<double>& lower, const std::vector<double>& upper, const std::vector<double>& velocity = {}, const std::vector<double>& acceleration = {})
```

Comment:
Those default values `= {}` are no longer needed (and also better placed in the header than in the implementation file)